src/media/js/addon/reducers/review.js

```diff
@@ -1,5 +1,5 @@
 /*
-  Add-ons in the review queue, keyed by slug.
+  Add-ons in the review queue, paginated, keyed by slug.
```

Comment:
It's not very clear what this object would look like, and `initialState` doesn't really help that. A quick example would be helpful.